### ISSUE TITLE
ShipperHQ sets Order to pending

### ISSUE THREAD

**Issue by beyondmediadev** [shipperhq/module-shipper#75]
Hi there,

we are working with PayOne in one of our shops. PayOne assigns the state processing when the amount has been paid. Somehow randomly (3 days later) the order is set to pending with following notice:
"ShipperHQ Notice: No detailed shipping information recorded" 

Why is that? We have shipperHq Autocomplete Extension installed but don't use the m2 integration.

# Expected Behavior
With shipperHQ Extension being disabled in Admin, no action will be taken by shipperHQ 

# Current Behavior
Orders get updated to status pending randomly. Order was previously in state processing

**Comment by wsajason** [comment]
Hi,
The module is not able to affect the order status as it is designed only to be used with shipping rate calculations; post-order is left to built-in functionality.
On the order the module checks whether the order number is present in the advanced order details tables (which stores details such as dimensional shipping boxes, dates, origins, etc). If details are present they will be displayed on the order. If not present an order comment is added (which is not sent to the customer) stating that no detailed shipping information was recorded. This flag denotes to the module that it should not check for order details unique to ShipperHQ (I.e.: dimensional shipping boxes, dates, origins, etc).

In short, the order comment is for reference only; no changes are made to the order itself.

**Comment by WSAlewis** [comment]
Hello @lbajsarowicz,

Than you for the detailed response.

I'll be raising this to our development team to assess further. We'll reach out once we have more information to share.

If you have any further questions or concerns let us know.
